ui/nuxeo-user-group-management/nuxeo-create-group.js

```diff
@@ -93,7 +93,7 @@ import '../nuxeo-button-styles.js';
 
         <div class="header">
           <iron-icon icon="nuxeo:group"></iron-icon>
-          <h3>[[i18n('createGroup.heading')]]</h3>
+          <h3 class="headline-4">[[i18n('createGroup.heading')]]</h3>
```

Comment:
No need to applying the `headline-4` class here, as we could just switch this `h3` to an `h4`.